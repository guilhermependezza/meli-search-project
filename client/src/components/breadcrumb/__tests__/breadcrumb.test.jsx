import React from 'react';
import { create } from 'react-test-renderer';

// Component
import Breadcrumb from 'components/breadcrumb';

describe('Header', () => {
  it('should match snapshot', () => {
    const categories = ['The', 'categories'];
    const breadcrumb = create(<Breadcrumb categories={categories} />);
    expect(breadcrumb).toMatchSnapshot();
  });
});
