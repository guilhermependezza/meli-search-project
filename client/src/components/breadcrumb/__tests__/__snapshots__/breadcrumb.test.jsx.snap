// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div
  className="breadcrumb"
>
  <span
    className="breadcrumb-item"
  >
    The
  </span>
  <span
    className="breadcrumb-item"
  >
    categories
  </span>
</div>
`;
